.husky/commit-msg

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx --no-install commitlint --edit "$1"
+npx --no -- commitlint --edit 
```

Comment:
Could you elaborate on the need for this change?

Reply:
It allows to easily set up Git hooks in your project as a pre-commit hook to enforce commit message linting using commitlint before allowing a commit to be made. I played around with it and it can be deleted. My bad :)